src/data/finsupp/defs.lean

```diff
@@ -955,6 +965,11 @@ lemma map_range_sub [sub_neg_zero_monoid G] [sub_neg_zero_monoid H]
   map_range f hf (v₁ - v₂) = map_range f hf v₁ - map_range f hf v₂ :=
 ext $ λ _, by simp only [hf', sub_apply, map_range_apply]
 
+lemma map_range_sub' [add_group G] [subtraction_monoid H] [add_monoid_hom_class β G H]
```

Comment:
Should this be
```suggestion
lemma map_range_sub' [sub_neg_zero_monoid G] [sub_neg_zero_monoid H] [add_monoid_hom_class β G H]
```

Reply:
I think `map_sub` takes in `add_group` and `subtraction_monoid`.